viewing_party/main.py

```diff
@@ -0,0 +1,247 @@
+
+def create_movie(title, genre, rating):
+    """
+    inputs: title (str), genre (str), rating (num)
+    outputs: a dictionary with 3 key-value pairs, which each correspond to one of the parameters
+    """
+
+    # initialize a dictionary
+    movie_dict = {}
+
+    # if ALL parameter values are truthy, add keys and values to dictionary and return dictionary
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+
+    # if ANY of the parameters return falsy, the function should return None
+    return None
+
+def add_to_watched(user_data, movie):
+    """
+    inputs: user_data (a dictionary with an empty list as its value) and movie (dictionary)
+    outputs: returns the updated user_data dictionary
+    """
+
+    user_data["watched"].append(movie)
+
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    """
+    inputs: user_data (a dictionary with an empty list as its value) and movie (dictionary)
+    outputs: returns the updated user_data dictionary
+    """
+
+    user_data["watchlist"].append(movie)  
+
+    return user_data
+
+def watch_movie(user_data, title):
+    """
+    inputs: user_data (a dictionary with a "watchlist" and a "watched") and title (str)
+    outputs: return user_data dict, which will be updated depending on whether the user has watched that title
+    """
+    #print(user_data)
+
+    # if title is in watchlist, remove dictionary for that title from watchlist and add to watched list     
+    for each_movie in user_data["watchlist"]:
+        if each_movie["title"] == title:
+            # add that dictionary to watched list
+            user_data["watched"].append(each_movie)
```

Comment:
Consider using `add_to_watched`
```suggestion
            add_to_watched(user_data, each_movie)
```